### PR TITLE
Development grid stack

### PR DESCRIPTION
**Pull request by Mik-TF**
# Related Issue

- #655 

# Work Done

- Updated the full vm grid stack guide as requested in the issue mentioned above

### CODE REVIEW

**Comment by Mik-TF** [comment]
@A-Harby LMK if it's ok! Some stuff Bert will update on the grid_deployment's side, e.g. updating scripts to show if a file has been properly downloaded.

File: src/documentation/developers/grid_deployment/grid_deployment_full_vm.md [hunk 1]
```diff
@@ -128,11 +240,32 @@ Once you've prepared the VM, you can simply run the script to install the grid s
 sh install_grid_bknd.sh
 ```
 
-This will take some time since you are downloading the whole mainnet grid snapshots.
+This will take some time since you are downloading the whole mainnet grid snapshots. For mainnet, it will take several hours.
+
+You can also install the grid stack manually without snapshots. Depending on the available disk iops available, it can take up until a week to sync from block 0.
+
+```sh
+docker compose --env-file .secrets.env --env-file .env up -d
```
[the remark this answers is not in the document]
Comment (Mik-TF) [reply src/documentation/developers/grid_deployment/grid_deployment_full_vm.md:248]: OK I will remove that part then!

[the remark this answers is not in the document]
Comment (Mik-TF) [reply src/documentation/developers/grid_deployment/grid_deployment_full_vm.md:248]: Done! Thanks.

**Comment by A-Harby** [comment]
Last thing, but could you just add a note anywhere before the script about the downloading and the extracting time of the 3 files, just to let the user know it takes a long time to download and extract each?


It takes hours with nothing shown, so the note is very necessary to let the user know before starting the script. 
image

**Comment by Mik-TF** [comment]
> Last thing, but could you just add a note anywhere before the script about the downloading and the extracting time of the 3 files, just to let the user know it takes a long time to download and extract each?
> 

Is this sufficient?

image

File: src/documentation/developers/grid_deployment/grid_deployment_full_vm.md [hunk 2]
```diff
@@ -120,19 +231,41 @@ ufw enable
...
-Once you've deployed the grid stack online, you can access the different grid services by usual the usual subdomains:
+Once you've deployed the grid stack online, you can access the different grid services by usual the usual subdomains.
+
+The .secrets.env file contains a DOMAIN environment variable which is used in docker compose itself and inside several containers. After you deploy caddy will request several certificates for subdomains of your provided DOMAIN environment variable.
+Make sure the above DNS requirements are met, IPv6 is optional but we strongly encourage to configure it by default.
+
+These subdomains will be generated, for which Caddy will request a certificate for *.your.domain. 
 
 ```
 dashboard.example.com
```
Comment (A-Harby) [inline src/documentation/developers/grid_deployment/grid_deployment_full_vm.md:271]: Are these links actually correct?

image

Comment (Mik-TF) [reply src/documentation/developers/grid_deployment/grid_deployment_full_vm.md:271]: As @coesensbert explained to me once, this is normal. You need to use e.g.:

- https://metrics.harby.gridtesting/caddy
- https://metrics.harby.gridtesting/metrics

Comment (Mik-TF) [reply src/documentation/developers/grid_deployment/grid_deployment_full_vm.md:271]: Updated in latest commit.

Comment (coesensbert) [reply src/documentation/developers/grid_deployment/grid_deployment_full_vm.md:271]: > Member

please read the docs: https://github.com/threefoldtech/grid_deployment/tree/development/docker-compose#metrics

Comment (Mik-TF) [reply src/documentation/developers/grid_deployment/grid_deployment_full_vm.md:271]: I think at this point I will just create a new PR with this;

- the manual will link to the grid deployment repo docs. So we can focus on one set of docs. Grid deployment docs are more complete at this point

**Review by A-Harby — APPROVED** [review]
LGTM, Good job!

**Pull request merged**